Dockerfile

```diff
@@ -0,0 +1,69 @@
+####
+# This Dockerfile is used to build the container for mrrc uploader
+#
+# mrrc uploader requires python 3
+#
+# 0. Step into the project dir
+#
+# 1. Build the image
+#   docker/podman build -t mrrc-uploader:1.0.0 .
+#
+# 2. Run the container as daemon, mount the host ~/upload/ path to container /root/upload/ path,
+#   the uploading path is the dir location where you will upload the tarballs from
+#   add -e to set specific environment variables, such as: AWS_PROFILE, aws_endpoint_url, bucket
+#   docker/podman run -dit -v ~/upload/:/root/upload/ --name mrrc-uploader mrrc-uploader:1.0.0
+#
+# 3. Execute the container
+#   docker/podman exec -it mrrc-uploader bash
+#
+# 4. Start using uploader
+#   mrrc upload/delete from /root/upload/...
+###
+
+# parser directive, always points to the latest release of the version 1 syntax,
+# automatically checks for updates before building, making sure using the most current version
+# syntax=docker/dockerfile:1
+FROM python:3.8
+
+USER root
```

Comment:
We should always avoid to use root if possible. Maybe this can help: https://medium.com/@DahlitzF/run-python-applications-as-non-root-user-in-docker-containers-by-example-cba46a0ff384

Reply:
@ligangty ok, done.